contracts/access/PermitControl.sol

```diff
@@ -131,8 +131,8 @@ abstract contract PermitControl is Ownable {
     address _address,
     bytes32 _circumstance,
     bytes32 _right
-  ) public view returns (uint256) {
-    return permissions[_address][_circumstance][_right];
+  ) public view returns (bool) {
+    return permissions[_address][_circumstance][_right] > block.timestamp;
```

Comment:
Is there a reason why we made this change, i.e. gas efficiency reasons? It seems to me like we're otherwise just losing some functionality (the ability to ask when a right expires without having to directly check storage)?

If we do keep this:
1. `hasRightUntil` should be renamed to `hasRight` because it no longer supports the timestamp to check against for "until" semantics with appropriate updates to the NatSpec comments.

Reply:
We can have another function called `hasRight` which returns if permit time expired and keep old function as well. The change was to get rid of multiple nested checks in `Super1155`, `Super721` and `Super721IMX`, which were costly bytecode vise.

Reply:
> 
> 
> We can have another function called `hasRight` which returns if permit time expired and keep old function as well. The change was to get rid of multiple nested checks in `Super1155`, `Super721` and `Super721IMX`, which were costly bytecode vise.

Great, let's do that. :)